src/primitives/relocatable.ts

```diff
@@ -8,6 +8,20 @@ import {
 
 export type MaybeRelocatable = Relocatable | Felt;
 
+export namespace MaybeRelocatable {
+  export function isFelt(
+    maybeRelocatable: MaybeRelocatable
+  ): maybeRelocatable is Felt {
+    return maybeRelocatable instanceof Felt;
+  }
+
+  export function isRelocatable(
+    maybeRelocatable: MaybeRelocatable
+  ): maybeRelocatable is Relocatable {
+    return maybeRelocatable instanceof Relocatable;
+  }
+}
+
```

Comment:
Yes I agree about creating another module, I just did it and put the two functions in it.

However, very soon we will need to implement new methods for `MaybeRelocatable` such as arithmetic operations or comparisons. So very soon function imports will become too heavy for `MaybeRelocatable` because there will be too many of them and it will be more practical to create a dedicated class in the module. But we will do that as a follow up if that suits you.

Reply:
Indeed, when the module will grow we'll refactor it
We could use wildcard import to avoid specific function imports but making a dedicated class might be a better option here